src/lib/FieldType/Keyword/KeywordStorage/Gateway/DoctrineStorage.php

```diff
@@ -134,7 +134,7 @@ protected function getAssignedKeywords(int $fieldId, int $versionNo): array
     }
 
     /**
-     * Retrieves the content type ID for the given $fieldDefinitionId.
+     * Retrieves the Content type ID for the given $fieldDefinitionId.
```

Comment:
inconsistency, usually a "Content" starts with capital letter only at the beginning of a sentence or parameter description:
```suggestion
     * Retrieves the content type ID for the given $fieldDefinitionId.
```